### PR TITLE
Update serverlessservice_validation.go

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
Fixed a typo.

### CODE REVIEW

File: pkg/apis/networking/v1alpha1/serverlessservice_validation.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (spec *ServerlessServiceSpec) Validate(ctx context.Context) *apis.FieldErro
 		return apis.ErrMissingField(apis.CurrentField)
 	}
 	var all *apis.FieldError
-	// Spec mode must be from the enum and
+	// Spec mode must be from the enum
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation.go:39]: Remove trailing whitespace:
```suggestion
	// Spec mode must be from the enum
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: pkg/apis/networking/v1alpha1/serverlessservice_validation.go (hunk 1 above)
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation.go:39]: Format Go code:
```suggestion
	// Spec mode must be from the enum
```

**Comment by NEERAJAP2001** [comment]
@googlebot I signed it!

**Pull request merged**